src/main/java/org/mozilla/jss/CryptoManager.c

```diff
@@ -22,7 +22,7 @@
 #include <jssver.h>
 
 #include "pk11util.h"
-
+#include <Algorithm.h>
```

Comment:
Generally I like the idea of using angle brackets for system libraries and double quotes for internal (JSS) libraries but this hasn't always been followed. I'll leave it up to you though.